packages/http/lib/http-decorators.tsp

```diff
@@ -14,7 +14,7 @@ model HeaderOptions {
   /**
    * Determines the format of the array if type array is used.
    */
-  format?: "csv";
+  format?: "csv" | "multi" | "tsv" | "ssv" | "pipes" | "simple" | "form";
```

Comment:
I don’t think multi or form makes sense for header. I think the only one supported in openapi is csv(2.0) simple(3.x). Feel like we shouldn’t add extra options here

Reply:
Issue #1710 says we should. @markcowl can you comment?

Reply:
I am a bit confused by that issue. As it originally says to add support for openapi3 which was already done as part of the linked issue. 
so this seems just about adding the other terminology which everyone didn’t seem to be fully agreeing on.

Reply:
Sorry, I mean issue #1711.

Reply:
I think there is a difference between what is supported in OpenAPI2, OpenAPI3, and http itself.  I think in this library, we should be as inclusive as possible, and we may add filters or linting rules to limit usage where, say, http allows something that is not supported by OpenAPI3.

The reason for this issue comment is not to tie the values settable in the http library to values in OpenAPI2.

I think Timothee is right about making the corraltaion plain between csv/simple and multi/form extremely explicit.

If I could do anything here, I would convert this property into a string enum value, and deprecate 'csv' and 'multi' in favor of the new values.  But I don't think we can do that without a breaking change.

Reply:
I issue a warning for TSV, but if I do that for `csv` and `multi` it would also be considered a "breaking change" due to the `--warn-as-error` setting. I can put it in docs.

Reply:
I am ok with adding the openapi3 naming(as long as we do a good job to prevent confusion) for it but for header I don’t think there is in either openapi2 or 3 other values allowed except csv/simple

Reply:
Yes, but in http, it is allowed.  I think our existing linter rules in Azure.Core will cover this usage for Azure specs